PyramidInstallationWindows

```diff
@@ -0,0 +1,13 @@
+Install Python 2.7.14
+During installation, add python.exe to Path
```

Comment:
Is this an option during the installation? Like, is it installed via a wizard? Sorry, I have always installed python by way of the command line so this always has to be done manually afterward. Would it be possible to get download links and a screenshot to how part works? Want to try to make this as pain free as possible for Roger.

Reply:
Yes and yes. This is an option via the setup wizard. I can definitely get those links and a screenshot.